safe/gis/vector/multi_buffering.py

```diff
@@ -122,8 +123,9 @@ def multi_buffering(layer, radii, callback=None):
             callback(current=i, maximum=feature_count, step=processing_step)
 
     # We transfer keywords to the output.
-    buffered.keywords = layer.keywords
+    buffered.keywords = KeywordIO().read_keywords(layer)
```

Comment:
Yes we want the keyword wizard to open automatically after your tool.
But this layer won't have keywords at the beginning, right ?

Hum, this algorithm will create a memory layer right ? Then you give this layer to the datastore, right ? Then keywords will be written automatically. Let me check your code again